src/bin/make_docs.js

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env node -r esm
+
+/* eslint-disable
+	node/no-unpublished-import,
+	node/shebang
+*/
+
+import path from 'path';
+import * as _ from 'lamb';
+import {readJson, saveObj} from '@svizzle/file';
+import {isKeyValue} from '@svizzle/utils';
+
+const DOCS_PATH = path.resolve(__dirname, '../node_modules/app/data/agg_docs.json');
+
+const PATH = path.resolve(__dirname, '../node_modules/app/data/typedoc_out.json');
```

Comment:
Minor, I'd rename this to `TYPEDOC_PATH` moving it before `DOCS_PATH` to communicate the flow of the process.